06-arrays.Rmd

```diff
@@ -0,0 +1,104 @@
+```{r, include = FALSE}
+ottrpal::set_knitr_image_path()
+```
+
+# Using Arrays For Parallelization and Other Use Cases
+
+We have a workflow that runs on a single sample. What if we want to process multiple samples at once? Let's look at the various ways we can run our workflow more efficiently, as well as processing many samples in parallel. This is where WDL really shines.
+
+- How to use scattered tasks to run a workflow on multiple samples at once
+
+- How to use arrays effectively
+
+- How to reference arrays in a task's command section
+
+- How arrays differ from Structs
```

Comment:
The purpose of this list confused me - I thought it was the outline of the chapter's sections, but it's not!

Reply:
I was thinking about this in terms of "learning objectives," not really a TOC. An older template had learning objectives so that's what I had it in for. Happy to remove it if you think it's unneeded.